src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1770,10 +1969,13 @@ class CgroupUtils(object):
                  repr(self.subsystems),
                  repr(self.paths),
                  repr(self.vntype),
-                 repr(self.assigned_resources)))
+                 repr(self.assigned_resources),
+                 repr(self.systemd_version)))
```

Comment:
There's too many arguments to the string, you need to add another `%s` to the format string.

Reply:
Fixed in next commit.